### ISSUE TITLE
The infinite loading example it's not working

### ISSUE THREAD

**Issue by Dagdelo** [tangbc/vue-virtual-scroll-list#184]
After scrolling to the bottom of the page, the loading spinner never ends.

**Comment by tangbc** [comment]
I try was working well, how do you meet this problem?

**Comment by Dagdelo** [comment]
I simple, tried to load more items on infinite scroll example...
![image](https://user-images.githubusercontent.com/10082704/79673367-788d2f80-81af-11ea-9371-4ecd408e0a5b.png)

**Comment by tangbc** [comment]
Look at your Console. is there no log out 'at bottom'?

This example scrolls to the real bottom to start adding data, but loading-spinner div always keep in bottom. Just use `bottom-threshold` equals to loading div height can solve.

**Comment by Dagdelo** [comment]
This solved!

**Comment by tangbc** [comment]
Maybe not trigger reach bottom, can you console.log these three const value when problem occur?

In source code here:
https://github.com/tangbc/vue-virtual-scroll-list/blob/master/src/index.js#L182

```js
console.log(offset, clientSize, scrollSize)
```

**Comment by Dagdelo** [comment]
Well, I've cloned the repo, installed packages, and serve the examples. This is my log
[localhost-1587252745892.log](https://github.com/tangbc/vue-virtual-scroll-list/files/4498040/localhost-1587252745892.log)

**Comment by tangbc** [comment]
I mean, in Chrome's console

**Comment by Dagdelo** [comment]
![image](https://user-images.githubusercontent.com/10082704/79674031-ad03ea00-81b5-11ea-9d6c-84a80aad13d0.png)

**Comment by tangbc** [comment]
By the way, these warnning maybe cause some bug:

```console
vue.runtime.esm.js?e832:619 [Vue warn]: $attrs is readonly.
found in
---> <VirtualList>
       <InfiniteLoading> at src/views/infinite-loading/Main.vue
         <App> at src/App.vue
           <Root>
warn @ vue.runtime.esm.js?e832:619
```

Reason is that you have two `node_modules` both contains `vue` dependencies in `package.json`, just remove one.

**Comment by tangbc** [comment]
So problem is find: 2181.60009 + 496 = 2677.60009, not equals to 2678, so I have to use `Math.ceil(offset)` to solve the problem thoroughly, I will publish a version to fix it.

**Comment by tangbc** [comment]
Published `v2.0.6`
